osi_account_bank_statement/models/account.py

```diff
@@ -29,27 +28,15 @@ def _domain_move_lines_for_reconciliation(self,
             acc_move_obj = self.env['account.move.line']
             journal = self.env['account.journal'].browse(self._context.get
                                                          ('journal_id', False))
-
-            if journal and journal.liability_account_id:
-                # Get account lines only for liable account
-                acc_ids = acc_move_obj.search(
-                    [('account_id',
-                      'in',
-                      journal.liability_account_id.id)]).ids
```

Comment:
@max3903 : Why are we removing this liability_account ?
We are changing the migration code then.
If we remove this field then there is no use of depended module 'account_payment_credit_card'.
As this field is populated from that module.
Thanks,
Mayank

Reply:
@mgosai This field does not exist:
https://github.com/OCA/account-payment/blob/12.0/account_payment_credit_card/models/account.py

and is not required: We already have the AP account to use on the credit card company record...